RHEL/7/input/remediations/bash/mount_option_tmp_noexec.sh

```diff
@@ -0,0 +1,14 @@
+# platform = Red Hat Enterprise Linux 7
+FSTAB=/etc/fstab
+SED=`which sed`
```

Comment:
Really don't need this variable here. `/usr/bin/sed` is fine. However if the variable is kept, this should be `SED=/usr/bin/sed`